docs/release/README.md

```diff
@@ -0,0 +1,56 @@
+# Tektoncd Operator Release
+
+## What is a Tektoncd Operator Release
+
+A release will consititue tagged Github release (release.yaml) and a release to operatorhub.io. At present, our goal is to make an operator releases at on a fixed schedule (eg: every 2 weeks) irrespective of the release cycle of other Tektoncd components (Pipelines, Triggers, Dashboard...). Each release of operator will bring the latest available version of each of the components at the time of the release.
```

Comment:
`s/will/is`. We should almost always use the present in those docs : we are stating what is a release (what we want it to be), not what it will be 😛